### PR TITLE
feat: handle auction bid success event

### LINKED ISSUE #612: Update auction price and auction owner for won LAND

### PR DESCRIPTION
**Pull request by NicoSantangelo**
requires a migrate up!

Closes #612

### CODE REVIEW

File: scripts/addAuctionTimestamp.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+UPDATE parcels SET auction_timestamp = 1517356800000 where auction_owner IS NOT NULL AND auction_price IS NOT NULL;
```
Comment (nachomazzara) [inline scripts/addAuctionTimestamp.sql]: Should we add this query to the migration up? how can we run it?

File: src/Asset/Asset.js [hunk 2]
```diff
@@ -31,18 +31,6 @@ export class Asset {
     return assets[0]
   }
 
-  findByIds(ids) {
```
Comment (nachomazzara) [inline src/Asset/Asset.js]: Why we are removing it?

File: src/Asset/Parcel/Parcel.model.js [hunk 3]
```diff
@@ -33,10 +33,6 @@ export class Parcel extends Model {
     return splitCoordinate(id)
   }
 
-  static findByIds(ids) {
```
Comment (nachomazzara) [inline src/Asset/Parcel/Parcel.model.js]: Why we are removing it?

File: src/Asset/Asset.js (hunk 2 above)
Comment (NicoSantangelo) [reply src/Asset/Asset.js]: Not used anywhere and was a security concern, ids was being interpolated and could be SQL injected

File: src/Asset/Parcel/Parcel.model.js (hunk 3 above)
Comment (NicoSantangelo) [reply src/Asset/Parcel/Parcel.model.js]: Not used anywhere and was a security concern, ids was being interpolated and could be SQL injected

File: scripts/addAuctionTimestamp.sql (hunk 1 above)
Comment (NicoSantangelo) [reply scripts/addAuctionTimestamp.sql]: you're right, historically we've been running `.sql` files directly on the server, but this should be done each time the migration runs, I'll change it 👍

**Pull request merged**